card/mod/05_standard/set/all/links.rb

```diff
@@ -14,6 +14,25 @@ def link_to text, href, opts={}
     end
   end
 
+  # link to url, view, card or related card
+  #
+  def smart_link link_text, target, html_args={}
```

Comment:
Probably not the best name choice. I needed a method that can create different link types (like card_link or view_link) depending on the argument so this method is 'smart' about creating the right link type. Maybe 'wagn_link' instead?